src/app/EventManagement.cpp

```diff
@@ -590,7 +598,14 @@ static bool IsInterestedEventPaths(EventLoadOutContext * eventLoadOutContext, co
     {
         if (interestedPath->IsEventPathSupersetOf(path))
         {
-            return true;
+            if (event.mFabricIndex.IsNull())
```

Comment:
Seems this logic should is not related to the path? Can we move this out side the loop?

Reply:
Done, thanks, It is ready for formal review.